### PR TITLE
Ensure New Network Info popup doesn't show when switching to testnets

### LINKED ISSUE #15854: [Bug]: Network popup appears for some short time when I change to a Testnet and then disappears
### Describe the bug

**Problem**: when I change to any of the test networks I can see that the Network popup appears for a short time and then disappears.

[Screencast from 15-09-22 21:07:04.webm](https://user-images.githubusercontent.com/54408225/190488796-23570bb0-b675-4acb-92cf-edd358449dcc.webm)

### Steps to reproduce

1. Unlock MM
2. Allow Testnets on the Advanced Settings
3. Change to any test network

### Error messages or log output

_No response_

### Version

10.19.0

### Build type

_No response_

### Browser

Chrome

### Operating system

Linux

### Hardware wallet

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by danjm**
Fixes https://github.com/MetaMask/metamask-extension/issues/15854

For a brief period of time when swtiching networks, the `getIsTestnet` selector returns undefined because the chainId is undefined. This PR ensures the New Network Info popup does not display if the chainId is undefiend.

### CODE REVIEW

**Comment by seaona** [comment]
The fix is working great @danjm !! I just saw that for Sepolia, we do show the popup, and maybe we shouldn't, as it is a testnet? I can open a separate issue if needed

image

File: ui/pages/routes/routes.component.js [hunk 1]
```diff
@@ -366,6 +367,7 @@ export default class Routes extends Component {
       isNetworkUsed,
       allAccountsOnNetworkAreEmpty,
       isTestNet,
+      currrentChainId,
```
Comment (adonesky1) [inline ui/pages/routes/routes.component.js:370]:
```suggestion
      currentChainId,
```

rrrrrrr 😄

File: ui/pages/routes/routes.container.js [hunk 2]
```diff
@@ -46,6 +47,7 @@ function mapStateToProps(state) {
     isNetworkUsed: getIsNetworkUsed(state),
     allAccountsOnNetworkAreEmpty: getAllAccountsOnNetworkAreEmpty(state),
     isTestnet: getIsTestnet(state),
+    currrentChainId: getCurrentChainId(state),
```
Comment (adonesky1) [inline ui/pages/routes/routes.container.js:50]: in all the spots

**Comment by danjm** [comment]
@seaona I fixed the sepolia issue with the latest commit. @adonesky1 I've fixed the typo you identified

**Review by brad-decker — APPROVED** [review]

**Review by adonesky1 — APPROVED** [review]

**Pull request merged**